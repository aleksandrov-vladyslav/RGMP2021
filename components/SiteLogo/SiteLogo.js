import React from 'react';

const SiteLogo = () => {
  return ( 
    <div className="site-logo">
      <span className="site-logo__bold">netflix</span>
      <span> roulette</span>
    </div>
   );
}
 
export default SiteLogo;