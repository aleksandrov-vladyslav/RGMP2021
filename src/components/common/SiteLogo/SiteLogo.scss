@import '../../../styles/variables.scss';

.site-logo {
  color: $color-red;
  font-size: 22px;

  &__bold {
    font-weight: bold;
  }
}