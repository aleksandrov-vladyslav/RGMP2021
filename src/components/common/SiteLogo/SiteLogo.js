import React from 'react';

import './SiteLogo.scss';

const SiteLogo = () => {
  return ( 
    <div className="site-logo">
      <span className="site-logo__bold">netflix</span>
      <span> roulette</span>
    </div>
   );
}
 
export default SiteLogo;