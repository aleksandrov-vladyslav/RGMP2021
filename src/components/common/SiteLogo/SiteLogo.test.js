import React from 'react';
import { render } from '@testing-library/react';

import SiteLogo from './SiteLogo';

describe('SiteLogo', () => {
  it('render SiteLogo snapshot', () => {
    const { asFragment } = render(<SiteLogo />);

    expect(asFragment(<SiteLogo />)).toMatchSnapshot();
  })
});