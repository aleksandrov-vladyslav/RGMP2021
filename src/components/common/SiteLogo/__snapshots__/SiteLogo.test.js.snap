// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SiteLogo render SiteLogo snapshot 1`] = `
<DocumentFragment>
  <div
    class="site-logo"
  >
    <span
      class="site-logo__bold"
    >
      netflix
    </span>
    <span>
       roulette
    </span>
  </div>
</DocumentFragment>
`;
